EXILED/Exiled.Events/Patches/Fixes/FixStaminaBar.cs

```diff
@@ -0,0 +1,73 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="FixStaminaBar.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Fixes
+{
+    using Exiled.API.Features.CustomStats;
+    using HarmonyLib;
+    using Mirror;
+    using PlayerRoles.FirstPersonControl;
+    using UnityEngine;
+
+    /// <summary>
+    /// Fix for <see cref="CustomStaminaStat"/>.
+    /// </summary>
+    [HarmonyPatch(typeof(FpcStateProcessor), nameof(FpcStateProcessor.UpdateMovementState))]
```

Comment:
I am pretty sure there a way to not having to patch this event

Reply:
Should prob be transpiler but idk